service-base/src/main/java/org/eclipse/hono/service/tenant/BaseTenantService.java

```diff
@@ -114,24 +120,25 @@ protected void doStop(final Future<Void> stopFuture) {
     /**
      * Processes a tenant request message received via the Vertx event bus.
      *
-     * @param tenantMsg The message.
+     * @param tenantMsg The message. Must not be null.
```

Comment:
FMPOV the better option is to document a `NullPointerException` being thrown if the message is `null`.